web/api/v1/api.go

```diff
@@ -574,6 +585,130 @@ func (api *API) alertmanagers(r *http.Request) (interface{}, *apiError, func())
 	return ams, nil, nil
 }
 
+// AlertDiscovery has info for all alerts
+type AlertDiscovery struct {
+	AlertGroups []*AlertGroup `json:"alertGroups"`
+}
+
+// Alert has info for a alert
+type Alert struct {
+	Labels      labels.Labels `json:"labels"`
+	Annotations labels.Labels `json:"annotations"`
+	State       string        `json:"state"`
+	ActiveAt    *time.Time    `json:"activeAt,omitempty"`
+}
+
+// AlertGroup has info for alerts part of a group
+type AlertGroup struct {
```

Comment:
An alert group is something we have in the alertmanager. This is the output of an alerting rule.

Reply:
Good point. Should I rename this to `AlertingRule` or should we flatten (/denormalize) the endpoint and expose a list of alerts right away?

Reply:
Flattening is probably easiest.